libweb3jsonrpc/UnixSocketServer.cpp

```diff
@@ -111,14 +111,15 @@ void UnixDomainSocketServer::Listen()
 			DEV_GUARDED(x_sockets)
 				m_sockets.insert(connection);
 
-			std::thread handler([this, connection](){ GenerateResponse(connection); });
+			std::thread handler([this, connection](){ GenerateResponse(connection); CloseConnection(connection);});
 			handler.detach();
 		}
 	}
 }
 
 void UnixDomainSocketServer::CloseConnection(int _socket)
 {
+	shutdown(_socket, SHUT_RDWR);
```

Comment:
`shutdown` should immediately send the signal to the other side, while `close` would wait for regular timeouts. Not sure how this applies to unix sockets though.